src/component/SearchResults.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import Moment from "react-moment";
+
+function CreateRowTable({ customer }) {
```

Comment:
You can drop the verb in the component name here.
```suggestion
function RowTable({ customer }) {
```